docs/integrations/databases/Weaviate.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Weaviate"
+sidebarTitle: "Weaviate"
+---
+
+## Credentials
+
+Open the file named `io_config.yaml` at the root of your Mage project and enter weaviate required fields:
+
+```yaml
+version: 0.1.1
+default:
+  WEAVIATE_ENDPOINT: https://some-endpoint.weaviate.network
+  WEAVIATE_INSTANCE_API_KEY: YOUR-WEAVIATE-API-KEY
+  WEAVIATE_INFERENCE_API_KEY: YOUR-OPENAI-API-KEY
+  WEAVIATE_COLLECTION: collectionn_name
+```
+
+## Using Python block
+
+1. Create a new pipeline or open an existing pipeline.
+2. Add a data loader or data exporter with Template. Under "Databases" category you can
+ find the "Weaviate" template. Then use corresponding load and export functions.
+
+- data loader arguments:
+  - properties (List): list of columns to load.
+  - collection (str): name of the collection.
+  - Defaults to the name defined in io_config.yaml.
```

Comment:
should this be indented 1 more time?

Reply:
Currently it is intended with two spaces. I am previewing with [visual stuodio mdx preview](https://marketplace.visualstudio.com/items?itemName=xyc.vscode-mdx-preview) it looks like this: 
![222](https://github.com/mage-ai/mage-ai/assets/5386254/70935466-6808-4925-a8e9-e1846a58c0bb)

Reply:
oh i meant specifically the `Defaults to the name defined in ioi_config.yaml` line. it seems like it's describing the `collection` argument

Reply:
You are right! Good catch. Merged `Defaults ` into the same line as `collection`